dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -3602,6 +3603,7 @@ def test_correct_null_row_counts(self):
             }
         )
         profile = dp.StructuredProfiler(data, options=profiler_options)
+        self.assertTrue(profile.options.row_statistics.null_count.is_enabled)
```

Comment:
I think the counts intuitively speak to whether this is enabled. The tests in the options should cover this being set properly as well.

We just want to make sure when enabled, it does collect the values as expected.